src/storage/sstable.rs

```diff
@@ -168,7 +194,8 @@ impl SSTable {
             if prev.is_some() && &k == prev.as_ref().unwrap() {
                 continue;
             }
-            serde::serialize_kv(&k, &v, &mut file)?;
+            k.ser(&mut file)?;
```

Comment:
it might simplify some call-sites to expose a function in `serde` like this:

``` rust
serde::serialize_kv(k, v, &mut file) -> Result<int> {
  i = k.ser(file)?;
  i += v.ser(file)?;
  i
}
```

Reply:
Done